catalog-info.yaml

```diff
@@ -1,30 +1,25 @@
 # Backstage documentation
 # https://backstage.io/docs/features/software-catalog/descriptor-format/
 
-# !!! WARNING !!!
-# This is a template file with a number of fields that need to be filled before merging this to the default branch
 apiVersion: backstage.io/v1alpha1
 # Component, API, Template, Group, User, Resource, System, Domain, Location
-kind: unknown
+kind: API
 metadata:
   name: boilerplate-php 
-  description: unknown
+  description: An example app to get started with using the Contentful PHP SDK.
   annotations:
     github.com/project-slug: contentful/boilerplate-php 
-    contentful.com/service-tier: "unknown" #1, 2, 3, 4
+    contentful.com/service-tier: "1" #1, 2, 3, 4
```

Comment:
this is _definitely_ not tier 1 😅 (that's CDA delivery path **services** territory). I think this is likely tier 4 or `unknown`?

Reply:
yeah tier 4 for all services that aren't directly tied to delivery and management SLAs.

Reply:
but this isn't even a _service_ – so i'm thinking unknown? 🤔

Reply:
I'm going with unknown for now :)